package.json

```diff
@@ -43,6 +44,7 @@
     "eslint": "^6.5.1",
     "eslint-plugin-oclif": "^0.1.0",
     "globby": "^10.0.1",
+    "listr": "^0.14.3",
```

Comment:
This refactor aimed to remove use of Listr in favor of verbose logging. Pretty much just pass everything through to stdout instead of forcing things into loading tasks.